zero/client_server/client.py

```diff
@@ -171,20 +174,21 @@ async def _poll_data(self):  # pragma: no cover
                 resp_id, data = self._encoder.decode(frames)
                 self._resp_map[resp_id] = data
                 await self.peer1.send(b"")
-            except Exception as e:
-                logging.error(f"Error while polling data: {e}")
+            except Exception as exc:
+                logging.error("Error while polling data: %s", exc)
```

Comment:
f string is faster?

Reply:
https://github.com/Ananto30/zero/pull/34#discussion_r1160584800